helm/aws-operator/values.yaml

```diff
@@ -66,7 +66,8 @@ tenant:
     clusterDomain: cluster.local
     networkSetup:
       docker:
-        image: giantswarm/k8s-setup-network-environment:1f4ffc52095ac368847ce3428ea99b257003d9b9
+        image:
+          giantswarm/k8s-setup-network-environment:1f4ffc52095ac368847ce3428ea99b257003d9b9
```

Comment:
```suggestion
        image: giantswarm/k8s-setup-network-environment:1f4ffc52095ac368847ce3428ea99b257003d9b9
```

Reply:
FYI `ruamel.yaml` (the python module) did that automatically, obviously to prevent overly long lines